radio/src/storage/eeprom_rlc.cpp

```diff
@@ -826,6 +826,14 @@ bool eeLoadGeneral()
     }
   }
 
+  #if defined(PCBX7)
+  if (g_eeGeneral.variant == 0) {
+    TRACE("Pre release EEPROM detected, variant %d instead of %d for X7 radio. Loading anyway", g_eeGeneral.variant, EEPROM_VARIANT);
+    g_eeGeneral.variant = EEPROM_VARIANT;
```

Comment:
perhaps not needed as we write the unexpected shutdown bit just after. perhaps I am wrong though, an eeDirty wouldn't hurt